grid/recruitment/create-lead/maps/breezy-hr.suma

```diff
@@ -348,4 +380,50 @@ operation CheckCompanyExists {
       }
     }
   }
-}
\ No newline at end of file
+}
+
+operation UploadCV {
+  cv = args.lead.cv || {}
+
+  return if(cv.data === undefined || cv.data === null) {}
+
+  tmp = (cv.data.name = cv.fileName)
+  tmp = (cv.data.mimetype = cv.mimeType)
+
+  http POST "/v3/company/{ args.companyId }/position/{ args.lead.jobId }/candidate/{ args.leadId }/resume" {
+
+    request "multipart/form-data" {
+      headers {
+        Authorization = args.accessToken
```

Comment:
Why not use a security scheme?

Reply:
I agree. I suggest to change it in separete PR, as this is how all other Breezy maps work.